hl7-mllp/2.1/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,35 @@
+= HL7 MLLP Connector 2.1 - Mule 4
```

Comment:
Skimmed under assumption that boilerplate was applied.

Reply:
yes